### ISSUE TITLE
Deep Modification, issue with bulma

### ISSUE THREAD

**Issue by i13-ctflora** [StefMa/hugo-fresh#197]
After following the 'Deep Modification' steps, I'm getting an error:

`Error: error building site: TOCSS: failed to transform "style.sass" (text/x-sass): "/Users/christopher/code/hugo/fresh-test/themes/hugo-fresh/assets/style.sass:1:1": File to import not found or unreadable: bulma/bulma.`

I have updated the line in the themes/hugo-fresh/config.yaml file:
```
module:
  imports:
  - path: bulma
```

And I have updated the main hugo.yaml file to remove the 'module' section and add the theme entry:
`theme: 'hugo-fresh'`

The line in style.sass is the bulma import.  If I change it to `@import ../../bulma/bulma`, then hugo will compile but the css is all jacked up:
![image](https://github.com/StefMa/hugo-fresh/assets/157301727/d34ad6a8-2a9b-47f2-8605-495a2b9208d4)

Any ideas what I'm doing wrong here?

**Comment by i13-ctflora** [comment]
Turns out Bulma *just* released a new breaking version 1.0.0 (last week).  It's apparently a whole rewrite, and some of the mount targets referred to in the hugo-fresh config.yaml file no longer exist (bulma.sass is now bulma.scss, bulma-rtl.sass is gone or renamed & moved).  I reverted to the previous bulma version and everything works fine.

To revert, go into the themes/bulma folder and checkout the 0.9.4 version:
```
cd themes/bulma
git checkout 0.9.4
```
